src/fectch.js

```diff
@@ -0,0 +1,34 @@
+// import { url } from './url.js';
+
+// const setNewScore = async (newScore) => {
+//   //   const getId = await getGameId();
+//   try {
+//     const result = await fetch(
+//       'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/tJ0TIDemXqPa7nFliAbp/scores',
+//       {
+//         method: 'POST',
+//         body: JSON.stringify(newScore),
+//         headers: {
+//           'Content-type': 'application/json; charset=UTF-8',
+//         },
+//       },
+//     );
+//     const newScoreResult = await result.json();
+//     return newScoreResult;
+//   } catch (error) {
+//     return error;
+//   }
+// };
+
+// const getScores = async () => {
+//   //   const getId = await getGameId();
+//   const resultSaved = await fetch(
+//     'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/tJ0TIDemXqPa7nFliAbp/scores'
+//   );
+//   const savedScores = await resultSaved.json();
+//   console.log(savedScores.result);
+//   addscores(savedScores.result);
+// };
+// getScores();
+
+// export { setNewScore, getScores };
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. This will make your code much noisier and not professional, It is always a good practice to keep your code clean from such comments 🙃 😉